docs/english/web.md

```diff
@@ -51,6 +53,110 @@ response = client.chat_postEphemeral(
 
 See the [`chat.postEphemeral`](/reference/methods/chat.postEphemeral) API method for more details.
 
+### Sending streaming messages {#sending-streaming-messages}
+
+You can have your app's messages stream in to replicate conventional AI chatbot behavior. This is done through three Web API methods:
+
+* [`chat_startStream`](/reference/methods/chat.startstream)
+* [`chat_appendStream`](/reference/methods/chat.appendstream)
+* [`chat_stopStream`](/reference/methods/chat.stopstream)
+
+:::tip [The Python Slack SDK provides a [`chat_stream()`](https://docss.slack.dev/tools/python-slack-sdk/reference/web/client.html#slack_sdk.web.client.WebClient.chat_stream) helper utility to streamline calling these methods.]
+
+See the [_Streaming messages_](/tools/bolt-python/concepts/message-sending#streaming-messages) section of the Bolt for Python docs for implementation instructions. 
+
+:::
+
+#### Starting the message stream {#starting-stream}
+
+First you need to begin the message stream:
+
+```python
+# Example: Stream a response to any message
+@app.message()
+def handle_message(message, client):
+    channel_id = payload["channel"]
+    thread_ts = payload["thread_ts"]
+    
+    # Start a new message stream
+    stream_response = client.chat_startStream(
+        channel=channel_id,
+        thread_ts=thread_ts,
+    )
+    stream_ts = stream_response["ts"]
```

Comment:
```suggestion
def handle_message(message, client):
    channel_id = event.get("channel")
    team_id = event.get("team")
    thread_ts = event.get("thread_ts") or event.get("ts")
    user_id = event.get("user")
    
    # Start a new message stream
    stream_response = client.chat_startStream(
        channel=channel_id,
        recipient_team_id=team_id,
        recipient_user_id=user_id,
        thread_ts=thread_ts,
    )
    stream_ts = stream_response["ts"]
```

🪬 suggestion: Streaming into channel in response to messages outside of DMs will require a specific `recipient` at this time. Also added fallback for threading against top-level messages without a thread timestamp!

Reply:
+1 to this!